### PR TITLE
feat(draw): Management of the editing of the labels

### PR DESCRIPTION
**Pull request by kalvinkhuu**
**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

[Link to the issue](https://github.com/infra-geo-ouverte/igo2/issues/773)

**What is the new behavior?**

1. Added a cancel button with its respective logic
2. Allowing the user to double click the selected drawing to edit its label
3. User can now edit the font size and style of the labels of their drawings

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [X] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:

### CODE REVIEW

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]
I think the form-field for font type and size is a little bit big in the igo2 display. Maybe minimize it a bit and place them side by side ?

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 1]
```diff
@@ -61,21 +61,22 @@
...
+          (colorPickerChange)="onColorChange(labelsAreShown, false)"/>
       </mat-form-field>
     </div>
 
     <div class="stroke-color-picker mat-typography">
       <span>
-        <mat-icon class="stroke-palette-icon" svgIcon="square-outline"></mat-icon>
-        {{'igo.geo.draw.stroke' | translate}}
+        <mat-icon class="stroke-palette-icon" svgIcon="square-outline">
+        </mat-icon>
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:78]: Can stay on the same line (as before)

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 2]
```diff
@@ -85,7 +86,8 @@
         tooltip-position="below"
         matTooltipShowDelay="500"
         [matTooltip]="'igo.geo.draw.colorPicker' | translate">
-        <mat-label>{{strokeColor}}</mat-label>
+
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:89]: delete line

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 3]
```diff
@@ -95,44 +97,83 @@
...
           [cpPosition]="'bottom'"
           [cpPositionOffset]="'-75%'"
           [cpOutputFormat]="'rgba'"
           [cpCancelButton]="true"
-          [cpCancelButtonText]="'igo.geo.draw.cancelColorPicker' | translate"
+          [cpCancelButtonText]="'igo.geo.draw.cancel' | translate"
           [cpOKButton]="true"
-          (colorPickerChange)="onColorChange(labelsAreShown, false)">
+          (colorPickerChange)="onColorChange(labelsAreShown, false)"
+        />
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:108]: Can stay on the same line (as before). Remove "/"

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 4]
```diff
@@ -95,44 +97,83 @@
...
 
     <div>
       <mat-form-field *ngIf="icons.length >= 1">
-        <mat-label>{{'igo.geo.draw.icon' | translate}}</mat-label>
+        <mat-label>{{ 'igo.geo.draw.icon' | translate }}</mat-label>
         <mat-select>
           <mat-select-trigger>
             <div *ngIf="icon" class="box">
-              <img src={{icon}}>
+              <img src="{{ icon }}" />
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:118]: Remove "/"

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 5]
```diff
@@ -95,44 +97,83 @@
...
             </div>
           </mat-select-trigger>
-          <mat-option value="" (click)="onIconChange()">{{'igo.geo.draw.noIcon' | translate}}</mat-option>
+          <mat-option value="" (click)="onIconChange()"
+            >{{ 'igo.geo.draw.noIcon' | translate }}
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:122]: Can stay on the same line (as before)

File: packages/geo/src/lib/draw/draw/draw.component.html [hunk 6]
```diff
@@ -95,44 +97,83 @@
...
+          </mat-option>
           <mat-option
             *ngFor="let icon_html of icons"
             [value]="icon_html"
             (click)="onIconChange(icon_html)">
             <div class="box">
-              <img src={{icon_html}}>
+              <img src="{{ icon_html }}" />
```
Comment (PhilippeLafreniere18) [inline packages/geo/src/lib/draw/draw/draw.component.html:129]: Remove "/"

**Review by PhilippeLafreniere18 — CHANGES_REQUESTED** [review]

**Comment by pelord** [comment]
@PhilippeLafreniere18  1.12 is the target?

**Review by pelord — APPROVED** [review]

**Pull request merged**